src/Servers/test/FunctionalTests/ResponseCompressionTests.cs

```diff
@@ -32,7 +32,7 @@ public ResponseCompressionTests(ITestOutputHelper output) : base(output)
         }
 
         public static TestMatrix NoCompressionTestVariants
-            => TestMatrix.ForServers(ServerType.IISExpress, ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
+            => TestMatrix.ForServers(ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
```

Comment:
Only use comments so it's more obvious it needs to be reverted.
```suggestion
            => TestMatrix.ForServers(/*ServerType.IISExpress, */ ServerType.Kestrel, ServerType.Nginx,  ServerType.HttpSys)
```

Reply:
I don't think it matters. We are just going to do a straight revert on this no matter what.